SCOPE.md

```diff
@@ -0,0 +1,16 @@
+# Scope
+
+Specifies procedures for constructing and conveying artifact trees, directed acyclic graphs (DAGs), and other related 
+data structures for artifacts.  Including but not limited to:
+
+- formats for artifact identifiers
```

Comment:
I suspect we don't want to call out from our scope (which is hopefully relatively stable) to the website (where we may have more change).

Reply:
Right, we probably want to 
1. define terms in the spec rather than link to terms outside the spec, and 
1. only define them one time. 

So the link for defined terms should be to the [Spec Glossary](https://github.com/git-bom/spec/blob/main/SPEC.md#terms-and-definitions-mandatory). I believe the link above meets these requirements.

Reply:
Yes, works for me.